### DISCUSSION TITLE
Auto turn off desktop profile in pop-up menu is feature or bug?

### DISCUSSION THREAD

**Discussion by TiCoKH** [Valkirie/HandheldCompanion#1089] [Q&A] [answered]
Previously I get used to turn on use desktop profile and navigate mapped right analog as mouse in a pop-up menu.
In latest version I found desktop profile get disabled when pop-up menu get active and switch to controller based navigation on quick menu. I approve It's more straight forward on controller based handheld, I just needs to get used new way of navigation. But first I wasn't sure it is a feature or bug.

**Comment by CasperH2O** [comment, marked as answer]
This is as intended, for now. We will evaluate the UX long term.
